docs/user-guide.md

```diff
@@ -396,3 +396,10 @@ services:
 
 #### Warning about Deployment Config
 If the Docker Compose file has a volume specified for a service, the Deployment (Kubernetes) or DeploymentConfig (OpenShift) strategy is changed to "Recreate" instead of "RollingUpdate" (default). This is done to avoid multiple instances of a service from accessing a volume at the same time.
+
+#### Warning about change of Service name
+If the Docker Compose file has service name with `_` in it (eg.`web_service`), then it will be replaced by `-` and the service name will be renamed accordingly (eg.`web-service`).
+
+**We are doing this because "Kubernetes" doesn't allow `_` in object name.**
```

Comment:
This looks like another heading when it is rendered. Can you use different markup instead? 

Also, instead of "we are doing this", do you think "kompose does this" will be more appropriate?